kiwi/system/root_init.py

```diff
@@ -84,6 +85,8 @@ def create(self):
             data.sync_data(
                 options=['-a', '--ignore-existing']
             )
+            if Defaults.is_buildservice_worker():
+                copy('/.buildenv', self.root_dir)
```

Comment:
Anyway, even if it is just to be consistent with builds inside and outside of the buildservice, you got a point in the sense KIWI should always make use of it, at least, this way all images will be the same, regardless having buildservice involved or not.